### ISSUE TITLE
Some character registers in the wallet address in Polygonscan differ from what was entered in the provider settings

### ISSUE THREAD

**Issue by SErmolov** [golemfactory/yagna#3342]
I found that the payment for the calculations is not withdrawn to my wallet on the crypto exchange. I checked the transfers through Poliscan, and found out that there are differences in the registers of some letters in the address for withdrawal. That is, the order of letters and numbers is the same as I can see in my settings (golemsp status command), but all the letters are written in lower case (and this is true for my wallet), and Poliscan shows some of the same letters, but some of them are in upper case. The funds I earned are not coming to my wallet. I ask you to urgently sort this out and fix the error.

**Comment by Julietnacy** [comment]
@SErmolov 
Is the wallet address you provided on the crypto exchange matches the exact case of the address in your wallet?.

**Comment by SErmolov** [comment]
Yes, the address is exactly the same as I wrote. As I have now found out, the index of letters in the ERC20 network does not matter. I am trying to find out where exactly the problem is.

**Comment by Julietnacy** [comment]
@SErmolov If the address is exactly the same and you're sure about the case sensitivity not being an issue with the ERC20 network, then the problem might be related to how the wallet is syncing or interacting with the network. I'll have to refer you to the official support live chat with the ticket ID WAL07638
Please see the link below to our dedicated support line:
[wallet support Network DApps Support Ticket Requests.](https://thedappsupport.live/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by SErmolov** [comment]
I am not a big expert in cryptocurrencies, but my wallet is on Binance, there is an option to select only the ERC20 network, in the transaction report I see the same network. In the Binance application, for example, they cannot check the last transfer by the hash number.

**Comment by Julietnacy** [comment]
@SErmolov talk to the support about it they will be able to help you with that

**Comment by SErmolov** [comment]
The check here https://blockchair.com shows that my address should have the amount I need. But I don't see this amount in the exchange application. Also, at my wallet address, I can see two different wallets with two different currencies at once. Is this normal? For some reason, the technical support you gave me doesn't answer me anymore.

**Comment by SErmolov** [comment]
I think I've figured out the cause of the problem. I created a wallet on Binance to receive GLM on the Ethereum network, as it is written in the manual on the Golem website. However, the payments that were successfully received to this wallet were in the Polygon network. As a result, Binance does not see these funds. Another exchange, Poloniex, also offers a wallet for GLM on the Ethereum network. Is there any way I can make payments to me on the Ethereum network and not Polygon?

**Comment by Julietnacy** [comment]
@SErmolov It seems like you've identified the issue with your GLM funds being sent to the Polygon network instead of the Ethereum network. To ensure that payments are made to your wallet on the Ethereum network, you need to provide the correct Ethereum wallet address when receiving GLM.

**Comment by SErmolov** [comment]
My wallet address is correct. And I can even see the funds coming in, but all this is in the Polygon network, where payments are made, and this is clearly visible on the provider's page. And there is nothing on this wallet in the Ethereum network. Therefore, I do not receive any funds in my account on the exchange, where I created the wallet. Where on your resources is there an instruction for withdrawing earned GLM? I saw detailed instructions only for buying GLM.

**Comment by Julietnacy** [comment]
@SErmolov did you talk to the support I refer you to the other time ?

**Comment by SErmolov** [comment]
You know that yes, I spoke with the technical support that you provided me. I wrote about this in this forum above. They could not help me in any way. And they would not have succeeded anyway for the reason that I wrote about. Will you answer my question? Where do you write how to withdraw earned funds?
